src/components/entity/state-badge.ts

```diff
@@ -140,6 +140,9 @@ export class StateBadge extends LitElement {
         if (domain === "update") {
           hostStyle.borderRadius = "0";
         }
+        if (domain === "media_player") {
+          hostStyle.borderRadius = "10%";
```

Comment:
```suggestion
          hostStyle.borderRadius = "8%";
```

Reply:
To match our border radius usage in other places

Reply:
Good, can we merge?

Reply:
if you apply the suggestion 😄 👍